src/udev/udev-builtin-blkid.c

```diff
@@ -293,6 +298,9 @@ static int builtin_blkid(sd_device *dev, sd_netlink **rtnl, int argc, char *argv
         blkid_probe_set_superblocks_flags(pr,
                 BLKID_SUBLKS_LABEL | BLKID_SUBLKS_UUID |
                 BLKID_SUBLKS_TYPE | BLKID_SUBLKS_SECTYPE |
+#if BLKID_SUBLKS_FSINFO
```

Comment:
Some CIs seem to dislike this. How about to use `#if defined(...)`?

Reply:
Thanks! Fixed with `#ifdef` as suggested below.